bots/jabber_mirror_backend.py

```diff
@@ -73,6 +78,7 @@ def zulip_to_jid(email, jabber_domain):
 
 class JabberToZulipBot(ClientXMPP):
     def __init__(self, jid, password, rooms):
+        # type: (JID, str, Any) -> None
```

Comment:
One way to figure out what this `Any` should be is to see where this function gets called and see what the third argument is at that call. (Hint: because it's the `__init__` function for a class, the function call is actually going to look like `JabberToZulipBot(x,y,z)`, where `x`,`y`,and `z` will be the values being passed in.)